common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -346,16 +390,18 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
   }
 
 
-  private def article() = {
-    val contentApiItem = contentApi()
-    val content = Content.make(contentApiItem)
-    Article.make(content)
+  /*
+  * Comments cleaner
+  */
+
+  "AmpEmbedCleaner" should "change the avatar img in a comment to be an amp-img" in {
+    val result = cleanDocumentWithCommentEmbed()
+    result.getElementsByTag("amp-img").size should be(1)
```

Comment:
Is there anything else we should be checking here that are mandatory for amp? E.g. height and width attributes are present